R/copy_number_vaf_plot.R

```diff
@@ -45,7 +45,7 @@ copy_number_vaf_plot = function(this_sample_id,
     chrom_order = c(1:22, "X")
```

Comment:
Do we want to add `library(GAMBLR.data)` here as well to accommodate for the internal call of `assign_cn_to_ssm`, similar to what we do when `get_gambl_metadata` is being called internally?

Reply:
On second thought, maybe this logic only applies if we're calling say a `get_` function to get data to be used in the example and internal calls of such functions are ok since the GAMBLR.data is a dependency of GMABLR.viz.